src/main/java/com/cloudbees/jenkins/plugins/BitbucketHookReceiver.java

```diff
@@ -39,16 +41,21 @@ public String getUrlName() {
      */
     public void doIndex(StaplerRequest req) throws IOException {
         String body = IOUtils.toString(req.getInputStream());
```

Comment:
I know this is not part of this change, but I think this `toString` should be honouring the content char encoding.